live_test_request_using_Fetch_api.js

```diff
@@ -0,0 +1,47 @@
+const ERC20Abi = require("./ERC20Abi");
```

Comment:
This is just a simple script you can run from the cmd line to test the new functionality out with live chain data.